### PR TITLE
In test "test_set_as_non_admin" Hostname is not required to match error message

### PR DESCRIPTION
**Pull request by birbalsain**
#### Describe Bug or Feature
 Hostname is not required to match error message because test purpose is, can a hook be set by non admin user or not?.

#### Describe Your Change
 Removed the hostname from error message string.

#### Attach Test and Valgrind Logs/Output

[non_admin_user.txt](https://github.com/PBSPro/pbspro/files/4095917/non_admin_user.txt)

### CODE REVIEW

File: test/tests/functional/pbs_server_periodic_hook.py [hunk 1]
```diff
@@ -391,8 +391,8 @@ def test_set_as_non_admin(self):
         host_name = str(self.server.hostname)
         self.server.create_hook(hook_name, attrs={'enabled': "False"})
         attrs = {'event': "periodic", 'freq': '120'}
-        match_str1 = str(TEST_USER1) + "@" + host_name + \
-            " is unauthorized to access hooks data from server " + host_name
+        match_str1 = "unauthorized to access hooks data from server " + \
+            host_name
```
Comment (visheshh) [inline test/tests/functional/pbs_server_periodic_hook.py]: host_name is used here

Comment (birbalsain) [reply test/tests/functional/pbs_server_periodic_hook.py]: this hostname is server  hostname. We are not using client hostname.

**Comment by bhroam** [comment]
I see you updated your PR title, could you update your git commit title as well?

File: test/tests/functional/pbs_server_periodic_hook.py (hunk 1 above)
Comment (vstumpf) [reply test/tests/functional/pbs_server_periodic_hook.py]: This is the wrong hostname

**Comment by birbalsain** [comment]
@visheshh @vstumpf I have done the requested changes.

**Comment by birbalsain** [comment]
@PBSPro/pbspro-maintainers Please take this PR further.

**Comment by prakashcv13** [comment]
merged based on reviews from @bhroam, @vstumpf and @visheshh

**Pull request merged**